README.md

```diff
@@ -30,11 +30,19 @@ To create a hash chain, the arguments are:
 ```shell
 $ ./hashchain create ALGORITHM LENGTH SEED
 ```
+or 
+```shell
+$ ./hashchain create ALGORITHM OFFSET SIZE SEED
+```
 
 Simple example:
 ```shell
 $ ./hashchain create sha256 10 "my secret password"
 ```
+or equivalently,
+```shell
+$ ./hashchain create sha256 1 10 "my secret password"
```

Comment:
I think the naming `OFFSET` may not be really proper here.
If the value `OFFSET` starts from 1, then it feels much more like `INDEX` but not `OFFSET`.
If it is `OFFSET`, then I think the value of `OFFSET` should start from `0`